src/main/java/com/ericsson/ei/controller/SubscriptionControllerImpl.java

```diff
@@ -56,6 +57,7 @@
 
     private static final String SUBSCRIPTION_NOT_FOUND = "Subscription is not found";
     private static final String SUBSCRIPTION_ALREADY_EXISTS = "Subscription already exists";
+    private static final String INVALID_USER = "Invalid user! You must be logged in as the creater to a subscription modify it.";
```

Comment:
creater -> creator

Reply:
'You must be logged in as the creator of a subscription to modify it'

Reply:
Done.